<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign Modifiers</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabwidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="beams">
      <attribute name="title">
       <string>Beams</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="beam_area_checkbox">
         <property name="text">
          <string>Area</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="beam_area_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="beam_as2_checkbox">
         <property name="text">
          <string>As2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="beam_as2_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="beam_as3_checkbox">
         <property name="text">
          <string>As3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="beam_as3_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="beam_torsion_checkbox">
         <property name="text">
          <string>Torsion</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="beam_torsion_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="beam_i22_checkbox">
         <property name="text">
          <string>I22</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="beam_i22_spinbox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.350000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="beam_i33_checkbox">
         <property name="text">
          <string>I33</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="beam_i33_spinbox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.350000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="beam_mass_checkbox">
         <property name="text">
          <string>Mass</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="beam_mass_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="beam_weight_checkbox">
         <property name="text">
          <string>Weigth</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="beam_weight_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="columns">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="column_area_checkbox">
         <property name="text">
          <string>Area</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="column_area_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="column_as2_checkbox">
         <property name="text">
          <string>As2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="column_as2_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="column_as3_checkbox">
         <property name="text">
          <string>As3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="column_as3_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="column_torsion_checkbox">
         <property name="text">
          <string>Torsion</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="column_torsion_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="column_i22_checkbox">
         <property name="text">
          <string>I22</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="column_i22_spinbox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.700000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="column_i33_checkbox">
         <property name="text">
          <string>I33</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="column_i33_spinbox">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.700000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="column_mass_checkbox">
         <property name="text">
          <string>Mass</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="column_mass_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="column_weight_checkbox">
         <property name="text">
          <string>Weigth</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="column_weight_spinbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="apply_to_etabs_button">
       <property name="toolTip">
        <string>Export Selected Load Combinations To ETABS Model</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="toolTip">
        <string>Export Selected Load Combinations To ETABS Model</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="help">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset resource="../../civiltools.qrc">
         <normaloff>:/civiltools/images/help.png</normaloff>:/civiltools/images/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>beam_area_spinbox</tabstop>
  <tabstop>beam_as2_spinbox</tabstop>
  <tabstop>beam_as3_spinbox</tabstop>
  <tabstop>beam_torsion_spinbox</tabstop>
  <tabstop>beam_i22_spinbox</tabstop>
  <tabstop>beam_i33_spinbox</tabstop>
  <tabstop>beam_mass_spinbox</tabstop>
  <tabstop>beam_weight_spinbox</tabstop>
  <tabstop>beam_area_checkbox</tabstop>
  <tabstop>beam_as2_checkbox</tabstop>
  <tabstop>beam_as3_checkbox</tabstop>
  <tabstop>beam_torsion_checkbox</tabstop>
  <tabstop>beam_i22_checkbox</tabstop>
  <tabstop>beam_i33_checkbox</tabstop>
  <tabstop>beam_mass_checkbox</tabstop>
  <tabstop>beam_weight_checkbox</tabstop>
  <tabstop>column_area_spinbox</tabstop>
  <tabstop>column_as2_spinbox</tabstop>
  <tabstop>column_as3_spinbox</tabstop>
  <tabstop>column_torsion_spinbox</tabstop>
  <tabstop>column_i22_spinbox</tabstop>
  <tabstop>column_i33_spinbox</tabstop>
  <tabstop>column_mass_spinbox</tabstop>
  <tabstop>column_weight_spinbox</tabstop>
  <tabstop>column_area_checkbox</tabstop>
  <tabstop>column_as2_checkbox</tabstop>
  <tabstop>column_as3_checkbox</tabstop>
  <tabstop>column_torsion_checkbox</tabstop>
  <tabstop>column_i22_checkbox</tabstop>
  <tabstop>column_i33_checkbox</tabstop>
  <tabstop>column_mass_checkbox</tabstop>
  <tabstop>column_weight_checkbox</tabstop>
  <tabstop>apply_to_etabs_button</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>help</tabstop>
  <tabstop>tabwidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
